sdk/cosmos/azure-cosmos-spark_3-0_2-12/docs/quick-start.md

```diff
@@ -0,0 +1,129 @@
+## Quick Start Guide for Cosmos DB Spark Connector Preview
+This tutorial is a quick start guide to show how to use Cosmos DB Spark Connector Preview to read from or write to Cosmos DB. Cosmos DB Spark Connector is based on Spark 3.0.x. 
+
+Throughput this quick tutorial we rely on Azure DataBricks Spark 3.0.1 and JupyterNotebook to show how to use the Cosmos DB Spark Connector.
```

Comment:
For simplicity (which Databricks runtime version)

Reply:
Thanks. DONE